framework/tests/simple/README

```diff
@@ -0,0 +1,2 @@
+This directory holds one simple Java file used to start of checkers that test some initialization
```

Comment:
Grammar error here; I'm not sure of the right fix.  Also missing period at end of second sentence.